### DISCUSSION TITLE
After update blocks FBW A320neo

### DISCUSSION THREAD

**Discussion by Reisener** [flybywiresim/aircraft#6499] [Help]
Hello,
after today's update 1.21.18 I can fly all aircraft only not flybywire-aircraft-a320-neo.
I start the msfs2020 , then I select the FBW A320neo and the Airport . After that comes the blue bar at the bottom where it gets slower and slower to the end. When the blue bar is at the end the screen stays like this and the music continues. But it does not jump to the airport. I have the 0.7.3 stable version. 
This is also true when I have in the community folder ( store version) only the FBW A320 neo and nothing else. In the MSFS content manager the FBW is deleted.
What to do ??? I can close the MSFS only with CTRL+Alt+DEL.
With kind regards
DLH874 Bernd

Translated with www.DeepL.com/Translator (free version)

**Comment by tracernz** [comment]
The stable version is not yet compatable with the MSFS hotfix. A new stable version is being tested now. The development and experimental versions are already working.

**Comment by smuffy23** [comment]
Reisener I have exactly the same issue, I also tried the dev version and had the same result, if I take the fbw A320 out of community folder the game works fine but its the default plane.

**Reply by ruimirada** [reply]
The problem remains, does not start the flight, without it in the community folder everything works, nor with the version Stable V0.7.4 and Development

**Reply by smuffy23** [reply]
Same here ruimirada, I have tried a clean install of stable and dev versions, still no joy, really frustrating.

**Reply by ruimirada** [reply]
Really frustrating.

**Reply by smuffy23** [reply]
Just did a clean install a10 minutes ago16.30 of FBW a320NX stable 7.04. it worked, good job by the devs. Good luck.

**Reply by ruimirada** [reply]
Tanks for report

**Comment by Reisener** [comment]
Hello,
with me also everything runs again as before. Thanks again for the
quick help. Will probably happen more often with the MSFS2020 update. 

I have previously additionally in the Store version in the folder 
 LocalState - packages - flybywire-aircraft-a320-neo everything deleted except the folder work. The other files are generated again on restart. The first time the loading takes a little longer, then it goes faster again.
This trick I have read somewhere is a kind of "cache".
So thanks again.
With kind regards
DLH874 Bernd

Translated with www.DeepL.com/Translator (free version)

**Comment by smuffy23** [comment]
Your welcome.
________________________________

**Comment by franek233233** [comment]
I warmly welcome.
I deleted everything related to the Community directory, then reinstalled the A320neo FBW and it's the same.
So the blue bar is about to finish loading, but nothing is happening. Colleagues, help me please.

**Comment by smuffy23** [comment]
Hi, I did a new download of the FBW downloader and installed the A320 stable version 7.04 and that worked, hope you get it sorted.
________________________________

**Comment by Reisener** [comment]
I have previously additionally in the Store version in the folder
LocalState - packages - flybywire-aircraft-a320-neo everything deleted except the folder work. The other files are generated again on restart. The first time the loading takes a little longer, then it goes faster again.
This trick I have read somewhere is a kind of "cache".
So thanks again.
With kind regards

**Comment by franek233233** [comment]
Thanks a lot.
So far I deleted everything from the community and installed the development version and it went. Will try with the stable version later.
Best wishes.

**Comment by DedaL54** [comment]
" I deleted everything related to the Community directory, then reinstalled the A320neo FBW and it's the same.
So the blue bar is about to finish loading, but nothing is happening......  "
HI , 
It's exactly the same with me! Three days after the last update (1.21.18.0 of MSFS)  I'm trying everything I've read on the forums. It doesn't work and it doesn't work .... The A320NX stays in the hangar and has no chance of appearing anywhere on the map!
I would be happy if someone could explain the nature of the problem step by step, because the installer does not report any problems.
The last attempt was an hour ago .... I don't know, I'm very frustrated that I don't know what to do! Help me please.
Greetings !

**Comment by Reisener** [comment]
Who can read is clearly at an advantage.
Sorry, is that so hard to understand ???????
1. step community folder is empty.
2. in my post I am no longer talking about the community folder, but the folder of the MSFS2020/LocalState/packages/flybywire-aircraft-a320-neo.   Open directory, there is the folder work and other files in it.
Do not delete the folder " work ".  
Delete all other files like .cache, .dll .lib .natvis .pdb. 
These will be regenerated when you restart MSFS2020 and it will take a little longer. 
At the next start these remain then stored and the start goes clearly faster. That has helped with me to, more I do not know also.
MfG
DLH874 

Translated with www.DeepL.com/Translator (free version)

**Comment by DedaL54** [comment]
"Who can read is clearly at an advantage.
Sorry, is that so hard to understand ???????"

I see no reason for this tone, especially since English is not my language .... anyway, we are not all the same and we do not all understand equally professional terms ... I did everything as you wrote in steps 1 and 2 and again no results!
So here's what I did next : I erased everything I had. ( FlyByWire instaler & .... ) I directly installed the A32NX-stable WinRAR ZIP archive properly in the Community folder along a defined path( Steam version) and unpacked. I ran MSFS, the A320NX is in the hangar, I put it on the map and that’s it!
Landscape, the blue line is coming to an end and I can just have something to calm down ... there are no planes ! Everything else by default is fine. What else can be done? My knowledge is not computer science and that is why I wander and do what I read!
I am finished ! Thanks for the support but I don't know what to do next and who to ask for advice!
So thanks again. Greetings !

**Comment by Reisener** [comment]
Hello,
again a last try from me to help you. 
1. the cache should be deleted as described in my previous post.
2. the community order is completely empty for now.
3. start MSFS and in the content manager " the " A32NX from Flybywire " must be deleted. In my case it says not installed.
Then quit MSFS2020 and install only the version 0.7.4. stable with the newest FBW installer. 
Then start MSFS and select the A320 neo (LEAP) + at the airport and start. I hope that it then works for you.
As I said the first time the loading at the right end of the bar takes a little longer. After that, he has already saved the data per aircraft and loads faster.
MfG DLH Bernd 
Store Version
Hallo,
nochmal ein letzter Versuch von mir Dir zu helfen. 
1. Der Cache soll wie in meinem vorherigen Beitrag  beschrieben gelöscht werden.
2. Der Community Orden ist erstmal komplett leer.
3. MSFS starten und im Inhaltmanager " muß "  der " A32NX von Flybywire " gelöscht werden. Bei mir steht dann  dahinter nicht installiert.
Dann MSFS2020 beenden und über den neuesten FBW installer erstmal nur die Version 0.7.4. Stable installieren. 
Danach den MSFS starten und den A320 neo ( LEAP ) aussuchen + am Airport und starten. Ich hoffe das es dann bei dir auch geht.
Wie gesagt beim ersten mal dauert das laden am rechten Ende des Balkens etwas länger. Danach hat er die Daten je Flugzeug schon abgespeichert und lädt schneller.
MfG DLH Bernd 
Store Version

**Comment by DedaL54** [comment]
Vielen Dank ! Endlich hat es geklappt!

I did as you wrote, and it took a very long time (apr. 10 min.) to load . This time I was stubborn and waited and waited .... and finally! 
I almost gave up!
For now, everything is working properly ..... I will make a short flight right away! I'm happy now .... :)

Also, vielen Dank noch einmal!
Gruß !

**Comment by Reisener** [comment]
Ok,
it only takes so long the first time, after that it's faster.
Greetings
DLH874
